components/image.js

```diff
@@ -171,12 +177,16 @@ export default function ZoomableImage ({ src, srcSet, ...props }) {
   if (!src) return null
 
   if (trustedDomain) {
-    return (
-      <TrustedImage
-        src={src} srcSet={srcSet}
-        onClick={handleClick} onError={handleError} {...props}
-      />
-    )
+    if (me?.privates?.showImagesAndVideos !== false) {
```

Comment:
At this point it might be easier to read `!me || me.privates.showImagesAndVideos`.

Additionally, since I've seen above that you want to show images in the preview regardless of the setting, you also need to use it here else trusted images will not render in preview if the setting is enabled.

```suggestion
    if (props.tab === 'preview' || !me || me.privates.showImagesAndVideos) {
```

_afaik, we can assume `privates` exist_